test/parallel/test-timers-unenroll-unref-interval.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const common = require('../common');
+const timers = require('timers');
+
+const keepOpen = setTimeout(() => {
+  common.fail('Test timed out.');
+}, common.platformTimeout(500)); // Keep event loop open.
```

Comment:
> > All the unref timers are guaranteed to fire before this,

> No, they are not. 

Clarification: Yes, the unref'ed intervals will fire before the timer. However, the test will still fail if the immediates wrapped inside the unref'ed intervals don't fire before the timer, and that's what's not guaranteed.